src/api/goods.ts

```diff
@@ -10,10 +10,10 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort((a, b) => a.name.localeCompare(b.name)).slice(0, 5));
```

Comment:
you should handle errors in the `getAll` function

Reply:
i already handled it 
```
  return getAll()
    .then(goods => goods.sort((a, b) => a.name.localeCompare(b.name)).slice(0, 5));
```

Reply:
Where you handle error?
Where you `catch` an error?